src/Schedule.h

```diff
@@ -224,6 +258,44 @@ struct StorageDim {
     bool fold_forward;
 };
 
+/** This indicates two function stages which loopness are fused from outermost
```

Comment:
Perhaps: "This represents two stages with fused loop nests from outermost to a specific loop level.  The loops to compute func_1(stage_1) are fused with the loops to compute func_2(stage_2) from outermost to loop level var_name and the computation from stage_1 of func_1 occurs first."